code/api/src/modules/style/model.js

```diff
@@ -10,4 +10,12 @@ module.exports = function(sequelize, DataTypes) {
       type: DataTypes.STRING
     }
   })
+
+  Style.associate = function(models) {
+    Style.hasMany(models.User, {
```

Comment:
Dropping a note as a reminder to drop this if we don't end up using this association